Mathlib/MeasureTheory/Constructions/Prod/Integral.lean

```diff
@@ -153,8 +153,7 @@ theorem integrable_measure_prod_mk_left {s : Set (α × β)} (hs : MeasurableSet
   refine ⟨(measurable_measure_prod_mk_left hs).ennreal_toReal.aemeasurable.aestronglyMeasurable, ?_⟩
   simp_rw [HasFiniteIntegral, ennnorm_eq_ofReal toReal_nonneg]
   convert h2s.lt_top using 1
-  -- Porting note: was `simp_rw`
-  rw [prod_apply hs]
+  simp_rw [prod_apply hs]
```

Comment:
I don't have a strong opinion if this proof or its version prior to this PR is preferable. Same for the other change in this file.

Reply:
Agreed, this was probably too much unthinkingly changing everything back.
```suggestion
  rw [prod_apply hs]
```